### ISSUE TITLE
'quiet' option not available for Image generation

### ISSUE THREAD

**Issue by LouTerrailloune** [KnpLabs/snappy#243]
The 'quiet' option is not listed in the listed of available options, however it is available in the binary (at least on my version: wkhtmltoimage 0.12.1 (with patched qt))

**Comment by NiR-** [comment]
Indeed. Is snappy too verbose when generating images (a little example would be cool)?

**Comment by LouTerrailloune** [comment]
It's not about verbosity, it is about completeness (all wkhtmlto* should be available in the wrapper)

Without --quiet : 
```
Loading page (1/2)
[>                                                           ] 0%
[======>                                                     ] 10%
[=======>                                                    ] 12%
[=========>                                                  ] 15%
[==========>                                                 ] 18%
[============>                                               ] 20%
[=============>                                              ] 22%
[==============>                                             ] 24%
[=================>                                          ] 29%
[===================>                                        ] 33%
[======================>                                     ] 38%
[========================>                                   ] 40%
[=========================>                                  ] 42%
[==========================>                                 ] 44%
[===========================>                                
 ] 46%
[==============================>                             ] 50%
[==============================>                             ] 50%
[================================>                           ] 54%
[==================================>                         ] 58%
[====================================>                       ] 60%
[========================================>                   ] 67%
[==========================================>                 ] 70%
[============================================>               ] 74%
[=============================================>              ] 76%
[===============================================>            ] 79%
[=================================================>          ] 83%
QFont::setPixelSize: Pixel size <= 0 (0)
[===================================================>        ] 85%
[===================================================>        ] 86%
[===================================================>        ] 86%
[===================================================>        ] 86%
[===================================================>        ] 86%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 87%
[====================================================>       ] 88%
QFont::setPixelSize: Pixel size <= 0 (0)
QFont::setPixelSize: Pixel size <= 0 (0)
[====================================================>       ] 88%
[====================================================>       ] 88%
[====================================================>       ] 88%
[====================================================>       ] 88%
QFont::setPixelSize: Pixel size <= 0 (0)
[=====================================================>      ] 89%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
QFont::setPixelSize: Pixel size <= 0 (0)
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[======================================================>     ] 90%
[============================================================] 100%
Rendering (2/2)                                                    
[>                                                           ] 0%
[===============>                                            ] 25%
QFont::setPixelSize: Pixel size <= 0 (0)
[============================================================] 100%
Done                                                               
Exit with code 1 due to network error: ContentNotFoundError
```

With --quiet : 
```
QFont::setPixelSize: Pixel size <= 0 (0)
QFont::setPixelSize: Pixel size <= 0 (0)
QFont::setPixelSize: Pixel size <= 0 (0)
QFont::setPixelSize: Pixel size <= 0 (0)
Exit with code 1 due to network error: ContentNotFoundError
```

**Comment by NiR-** [comment]
Has been solved by #244.
